clipper_admin/clipper_admin/docker/docker_metric_utils.py

```diff
@@ -43,7 +43,8 @@ def run_query_frontend_metric_image(name, docker_client, query_name,
     :return: None
     """
 
-    query_frontend_metric_cmd = "--query_frontend_name {}".format(query_name)
+    query_frontend_metric_cmd = "--query_frontend_name {}:1337".format(
```

Comment:
Let's avoid hardcoding the port number. That port is defined here: https://github.com/ucbrise/clipper/blob/e6eebdc5c999a0cbd5213e5f4e150663cb41b6c4/clipper_admin/clipper_admin/container_manager.py#L5